content/en/resources/keps/_index.md

```diff
@@ -0,0 +1,22 @@
+---
+linktitle: Enhancements
+title: Kubernetes Enhancement Proposals (KEPs)
+description: |
+  List of Kubernetes enhancements
+type: docs
+menu:
+  main:
+    weight: 40
+---
+
+These data come from the [k/enhancements][k-enhancements]
+repository on GitHub.
+
+> Note: If you notice an empty cell in the table, the KEP may not have updated information.
+Please raise an issue in the [k/enhancements][k-enhancements] repository.
```

Comment:
If possible, avoid using a block quote to mark a note. Try this instead:

```suggestion

Reply:
```suggestion
{{% alert title="Note" color="info" %}}
If you notice an empty cell in the table, the KEP may not have updated information.
Please raise an issue in the [k/enhancements][k-enhancements] repository.
{{% /alert %}}
```

Reply:
Done.

Reply:
This is a nifty TIL for me. Thanks for the tip!